src/components/IOUConfirmationList.js

```diff
@@ -132,9 +132,15 @@ class IOUConfirmationList extends Component {
     /**
      * Gets splits for the transaction
      *
-     * @returns {Array}
+     * @returns {Array|null}
      */
     getSplits() {
+        // There can only be splits when there are multiple participants, so return early when there are not
```

Comment:
The changes in this file are just some cleanup that I noticed could have been done better.